src/back-end/lib/db/proposal/team-with-us.ts

```diff
@@ -587,6 +608,19 @@ export const readOneTWUProposal = tryDb<
     }
   }
 
+  // Read the members associated with the proposal from the db
+  const teamMembers =
+    getValidValue(await readTWUProposalMembers(connection, result.id), []) ??
+    [];
```

Comment:
Remove `?? []`

Reply:
though, removing it generates a `teamMembers is possibly null` error further down in the code.

Reply:
K easier to leave it then